packages/components/src/Lettera/Models.purs

```diff
@@ -633,3 +639,105 @@ editorialIdToUuid editorialId =
 url_namespace :: UUID.UUID
 url_namespace = unsafePartial $ fromJust $ UUID.parseUUID "6ba7b811-9dad-11d1-80b4-00c04fd430c8"
 
+encodeRequestHeaders :: Array AX.RequestHeader -> Maybe Json
+encodeRequestHeaders [] = Nothing
+encodeRequestHeaders hs =
+  let encodeHeader h obj = flip extend obj $ case h of
+        AX.Accept (MediaType v)      -> "Accept" := v
+        AX.ContentType (MediaType v) -> "ContentType" := v
+        AX.RequestHeader k v         -> k := v
+  in Just $ foldr encodeHeader jsonEmptyObject hs
```

Comment:
Is this going to include authorization headers in it? I think a better default would be to scrub them from `LetteraResponse` (and if convenient make it easy in the source to enable them for dev use when needed).